insights/parsers/lvm.py

```diff
@@ -34,6 +34,7 @@
 from .. import add_filter
 from .. import Parser, parser, get_active_lines, LegacyItemAccess
 from . import parse_fixed_table
+from insights.core.dr import SkipComponent
```

Comment:
Seems SkipException is more suitable than SkipComponent. See http://insights-core.readthedocs.io/en/latest/exception_model.html#skipcomponent-and-skipexception

Reply:
Below is the code in dr.py,  Only SkipComponent is pass exception. All other exceptions will be logged into log file. I do not want to log anything for this as this is normal for some lvm version does not support one or two fields.

~~~
       try:
            if component not in broker and component in DELEGATES and ENABLED[component]:
                log.info("Trying %s" % get_name(component))
                result = DELEGATES[component](broker)
                broker[component] = result
        except MissingRequirements as mr:
            if log.isEnabledFor(logging.DEBUG):
                name = get_name(component)
                reqs = stringify_requirements(mr.requirements)
                log.debug("%s missing requirements %s" % (name, reqs))
            broker.add_exception(component, mr)
        except SkipComponent:
            pass
        except Exception as ex:
            log.exception(ex)
            broker.add_exception(component, ex, traceback.format_exc())
~~~

Reply:
Yes